source/documentation/values/calculations.html.md.erb

```diff
@@ -161,42 +171,55 @@ CSS to unitless numbers:
   @debug calc(-infinity) < math.$min-number  // true
 <% end %>
 
-## `min()` and `max()`
+## [Calculation Functions]
+ 
+* Comparison Functions: [`min()`], [`max()`], and [`clamp()`]
```

Comment:
It would be good to have some brief prose here to explain what's being listed (such as "Sass parses the following functions as calculations").